de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,111 @@
+app-id: de.nagstamon.nagstamon
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: nagstamon
+rename-desktop-file: nagstamon.desktop
+rename-icon: nagstamon
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=.nagstamon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --filesystem=xdg-run/keyring
+  - --filesystem=home
```

Comment:
why do you need full home access.

Reply:
The config is in in ~/.nagstamon and users can configure individual sounds for alerts which they need file system access for.